src/views/Account/AccountView.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <section class="account-view" v-if="user">
+    <figure>
+      <img :src="'data:image/jpeg;base64,' + user.img" alt="Profile Picture" class="profile-picture">
+    </figure>
+
+    <ul>
+      <li v-if="!isEditingProfileInfo"><strong>Username:</strong> {{ user.username }}</li>
+      <li v-else><strong>Username:</strong> <input v-model="user.username"></li>
+      <li v-if="!isEditingProfileInfo"><strong>Email:</strong> {{ user.email }}</li>
+      <li v-else><strong>Email:</strong> <input v-model="user.email"></li>
+      <li v-if="!isEditingProfileInfo"><strong>Phone Number:</strong> {{ user.phoneNumber }}</li>
+      <li v-else><strong>Phone Number:</strong> <input v-model="user.phoneNumber"></li>
+      <li v-if="!isEditingProfileInfo"><strong>Address:</strong> {{ user.address }}</li>
+      <li v-else><strong>Address:</strong> <input v-model="user.address"></li>
+      <li v-if="isEditingProfileInfo">
+        <button class="small-button" @click="onDoneButtonClick">Done</button>
+        <button class="small-button" @click="onCancelButtonClick">Cancel</button>
+      </li>
+      
+      <li v-if="!isChangingPassword"><strong>Password:</strong> ********</li>
+      <li v-else>
+        <strong>Current Password:</strong> <input type="password" v-model="currentPassword">
+        <strong>New Password:</strong> <input type="password" v-model="newPassword">
+      </li>
+      <li v-if="isChangingPassword">
+        <button class="small-button" @click="onConfirmButtonClick">Confirm</button>
+        <button class="small-button" @click="onCancelButtonClick">Cancel</button>
+      </li>
+    </ul>
+
+    <button @click="onEditProfilePictureButtonClick">Edit Profile Picture</button> 
+    <input type="file" ref="fileInput" @change="onFileInputChange" style="display: none;"> 
+
+    <button @click="onEditProfileInfoButtonClick">Edit Profile Info</button>
+
+    <button @click="onChangePasswordButtonClick">Change Password</button> 
+  </section>
+  <p v-else>Loading...</p>
+</template>
+  
+<script>
+import axios from 'axios';
+import { jwtDecode } from "jwt-decode";
+
+export default {
+  name: 'AccountView',
+  data() {
+    return {
+      user: null,
+      tempUser: null,
+      isEditingProfileInfo: false,
+      isChangingPassword: false,
+      currentPassword: '',
+      newPassword: '',
+    };
+  },
+  async mounted() {
+    const token = localStorage.getItem('jwt');
+    const decodedToken = jwtDecode(token);
+    const userId = decodedToken.data.id;
```

Comment:
tijdens het inloggen krijg je ook de userId dus als je die dan in de auth store opslaat kan je er ook bij